spec/features/review_status_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+feature "filtrowanie recenzji po statusie" do
+  scenario "zarządzanie osobami bez uprawnień" do
+    visit '/people'
+
+    expect(page).to have_content 'Log in'
+  end
+
+  context "po zalogowaniu" do
+    include_context "admin login"
+
+     scenario "link do nowej recenzji" do
+      visit '/reviews/new'
+
+      expect(page).to have_css("#new_review input[value='Dodaj']")
+    end
+  end
+
+    scenario "tworzenie nowej recenzji" do
+      visit '/reviews/new'
```

Comment:
Ponieważ trzeba tylko przetestować filtrowanie, wystarczy aby w bazie były dwie recenzje. 
Nie trzeba odwiedzać kolejnych formularzy i dodawać recenzje. Wystarczy zrobić to tak jak w teście z filtrowanie osób - w bloku `before` utworzyć odpowiednie recenzje a potem je odfiltrować. Natomiast trzeba wziąć pod uwagę, że recenzja musi być powiązana z recenzentem i wersją artykułu, która to wersja musi być powiązana ze zgłoszeniem i redaktorem.